main.py

```diff
@@ -142,15 +146,15 @@ def log_crash(logtype, value, tb):
 #Version Number
 if game.settings['fullscreen']:
     version_number = pygame_gui.elements.UILabel(
-        pygame.Rect((1500, 1350), (-1, -1)), VERSION_NUMBER[0:8],
+        pygame.Rect((1500, 1350), (-1, -1)), get_version_info().version_number[0:8],
```

Comment:
consider storing this as a local variable to avoid calling the function multiple times

Reply:
https://github.com/Thlumyn/clangen/pull/850#discussion_r1133016585

Reply:
The function actually statically stores the VersionInfo instance anyways to avoid redundant disk access, so I guess it's just a matter of preference. There shouldn't be any overhead.